### ISSUE TITLE
[Question] Is KataGo 40b saturated?

### ISSUE THREAD

**Issue by y-ich** [lightvector/KataGo#558]
Hi.

Excuse me if it was already discussed on Discord.
(It is hard to follow DIscord's thread...)

Now the strongest confidently-rated network is kata1-b40c256-s9948109056-d2425397051, whose "s" number returns back to less than 10G.
Is 40b weight saturated?

**Comment by sbbdms** [comment]
> Regardless, we will also be switching to 60b before too long. If you or anyone else would be able to do some time-parity tests on 60b vs 40b, that would be a great help. It's been a few months since anyone has done such tests at a serious scale, at least as far as posting results to here or Discord.

Recently I finished a test between b60s457 and b40s1007, which is the strongest confidently-rated 60b/40b network before I started the test.
(1) In the test, the b60s457 plays with 8000 playouts/move, and the b40s1007 plays with 16000 playouts/move.
Though in my observation, under the same time parity, 40b network can play a little more than 2 times of playouts of 60b network, so this setting might be a little bit unfair to the 40b network.
(2) The test plays under some pairs of custom parameters, which is the final set of params from my custom test mentioned in https://github.com/lightvector/KataGo/issues/508.
Whether the final set of params is stronger than the current released params, is quite hard too assess --- The result may vary much between different networks. However its overall strength should not be worse than the current one?
(3) 100 match games played in total. The 60b network has a 53% winrate.
Considering 40b network can play a little more playouts in same parity, I think the current strength (Under the same time parity) between 40b and 60b network might be similar. However the 60b has greater potential than the 40b, so I guess switching to 60b should be considerable.
I will arrange my procedure, SGFs and the result of my test, maybe in this weekend. Please point out if there's any mistake in my test.
I am also seeking for more assessment to my final set of params. If the final set of params proved to be weaker in your further assessment, then you can also assess the 2nd/3rd/4th set of params. I wish that there's a set of params from the test helps.

**Comment by sbbdms** [comment]
Hi!

I have uploaded the data of my latest test in https://github.com/lightvector/KataGo/issues/508.
The data includes match games between b60s457 and b40s1007 which I mentioned above.
Please have a check. Thanks!

**Comment by dionren** [comment]
@sbbdms I think we should take more visits to evaluate 40b/60b, because if with more visits like 100K vs 200K, the situation maybe different.  And with 8x3090, 15s/hand will have 1M visits.   If you can do more tests, i'm willing donate gpus for u.

**Comment by sbbdms** [comment]
@dionren The data in https://github.com/lightvector/KataGo/issues/508 contains config file of the match, and commands of BayesElo in description file (To assess the elo). If you have enough resources to start the match with extreme high playouts per move, then you can use the config file with modifications below:

maxPlayouts0 = 200000
maxPlayouts1 = 100000

You may also change the "CUDA GPU settings" or "OpenCL GPU settings" to enable multiple GPUs for the match, where I have no experience...
Running such 100 games with 8x3090 might cost around 3~4 days in total.

**Comment by michito744** [comment]
@dionren ok.

(;KM[6.5]RU[Japanese]GM[1]SZ[19]GN[]PW[W]CP[]AP[Lizzie: 0.7.2]BR[]DT[2020-09-05]EV[]PB[B]RE[]PC[]TM[]WR[]CA[Shift_JIS];B[dp];W[pp];B[dc];W[pd];B[nc];W[ce];B[dh];W[fe];B[fc];W[cj];B[ee](;W[dg];B[cg];W[ed];B[dd];W[cf];B[bh];W[eh];B[di];W[bg];B[ch];W[de];B[ef];W[fd](;B[eg];W[ec];B[eb];W[fb];B[db];W[gc](;B[cd];W[bd];B[bc];W[fh];B[df];W[lc];B[ne];W[pf];B[ng];W[ph];B[jc];W[kd];B[ie];W[jd];B[id];W[ic];B[hc];W[ib];B[hb];W[jb];B[ff];W[gf];B[ge];W[fg];B[dg];W[he];B[hd];W[gd];B[hg];W[gg];B[hh];W[gb];B[ia];W[ja];B[ha];W[fj];B[hf];W[ge];B[kb];W[kc];B[ka];W[jc];B[kf];W[lf];B[kg];W[md];B[nd];W[mf];B[nf];W[lh];B[lg];W[mg];B[mh];W[nh];B[mi];W[ke];B[me];W[le];B[kh];W[jf];B[if];W[mc];B[pc];W[qc];B[qb];W[pb];B[oc];W[rb];B[qd];W[rc];B[oh];W[ni];B[pg];W[oi];B[og];W[li];B[mj];W[lj];B[mk];W[ji];B[hj];W[pi];B[qg];W[qe];B[ri];W[qj];B[rj];W[ql];B[rk];W[pm];B[jg];W[qh];B[rh];W[rf];B[rg];W[ob];B[pj];W[oj];B[rm];W[rl];B[sl];W[qn];B[rn];W[qo];B[rp];W[ro])(;B[gg];W[bc];B[bb];W[be];B[cc];W[ab];B[bd];W[ad];B[ac]))(;B[ff];W[gf];B[gg];W[hf];B[ei];W[ec];B[eb];W[fb];B[db];W[gc];B[bd];W[fh];B[fg]))(;W[ef];B[df];W[de];B[ed];W[dg];B[eg];W[cf];B[ff];W[eh];B[df];W[cg];B[di]))

#456 has a super hard mode game that KataGo can't solve properly.

**Comment by sbbdms** [comment]
The recent three 40b networks (s1031, s1033 and s1035) seem to have significant elo gain. Is it the result of dropping learning rate?
Also I would like to know if my data is read during these weeks...
